RunInEditHelper/Editor/RunInEditHelper.cs

```diff
@@ -4,6 +4,35 @@
 
 namespace Unity.Labs.SuperScience
 {
+    public static class MonoBehaviourExtensions
```

Comment:
Actually, this _isn't_ a conflict with the version in utils, somehow. Namespaces ftw?